cloudbuild.yaml

```diff
@@ -3,16 +3,49 @@ steps:
     args:
       - build
       - '--build-arg'
-      - 'DOCKER_FLOW_VERSION=${_VERSION}'
+      - DOCKER_FLOW_VERSION=${_VERSION}
       - '-t'
-      - '${_IMAGE_NAME}'
+      - ${_IMAGE_NAME}:${_IMAGE_VERSION}
       - .
-timeout: 1500s
+    id: Build
+  - name: gcr.io/cloud-builders/docker
+    args:
+      - push
+      - ${_IMAGE_NAME}:${_IMAGE_VERSION}
+    id: Push
+  - name: bash
+    args:
+      - '-c'
+      - echo "$$DEV_CONFIG" > /workspace/dev-config.yml && echo "$$INGRESS" > /workspace/infrastructure/k8s/templates/ingress.yaml
+    id: Get secrets
+    secretEnv:
```

Comment:
Can you please leave somewhere in Readme a not about how to properly set up this env at least manually? In case we would need to reproduce it or someone else would like to run their own cloud build? 
Thanks

Reply:
https://github.com/taskcluster/taskcluster/pull/5578/commits/fd47dc069d7ae6ec3d27de7853289fa04480c215#diff-5b9888d738c268a2fca37f35bd3ddf988ebcac318b536ea2be2b45a3ea676f32R356-R361